.circleci/config.yml

```diff
@@ -195,3 +194,348 @@ workflows:
+              environment:
+                CHROMATIC_APP_CODE: <<parameters.CHROMATIC_APP_CODE>>
+                CHROMATIC_PROJECT_TOKEN: <<parameters.CHROMATIC_PROJECT_TOKEN>>
+              name: Chromatic - accept all changes
+      test-exit-cleanly-on-changes:
+        description: |
+          Run a chromatic test on the current branch, exiting cleanly unless there are errors.
+
+          If you are using the chromatic integration (recommended) then you will want to let
+          chromatic handle reporting back the status of your snapshots. This command itself
+          will only fail if the tests failed to run. If they succeeded with changes this command
+          will succeed and chromatic will report back the number of changes.
+        parameters:
+          CHROMATIC_APP_CODE:
+            default: ""
+            type: string
+          CHROMATIC_PROJECT_TOKEN:
+            default: ""
+            type: string
+          BUILD_SCRIPT_NAME:
+            default: ""
+            type: string
+        steps:
+          - run:
+              command: yarn chromatic test --exit-zero-on-changes --build-script-name <<parameters.BUILD_SCRIPT_NAME>>
+              environment:
+                CHROMATIC_APP_CODE: <<parameters.CHROMATIC_APP_CODE>>
+                CHROMATIC_PROJECT_TOKEN: <<parameters.CHROMATIC_PROJECT_TOKEN>>
+              name: Chromatic - report changes to chromatic
+      test-fail-on-changes:
+        description: |
+          Run a chromatic test on the current branch and fails if there are changes.
+
+          Generally, you would want to let chromatic report on changes and have this command
+          only fail if there was an error running it or rendering a component.
+
+          If, however, you are not enabling the chromatic integration then you can use this to
+          fail a build when there are any changes to the UI.
+        parameters:
+          CHROMATIC_APP_CODE:
+            default: ""
+            type: string
+          CHROMATIC_PROJECT_TOKEN:
+            default: ""
+            type: string
+          BUILD_SCRIPT_NAME:
+            default: ""
+            type: string
+        steps:
+          - run:
+              command: yarn chromatic test --build-script-name <<parameters.BUILD_SCRIPT_NAME>>
+              environment:
+                CHROMATIC_APP_CODE: <<parameters.CHROMATIC_APP_CODE>>
+                CHROMATIC_PROJECT_TOKEN: <<parameters.CHROMATIC_PROJECT_TOKEN>>
+              name: Chromatic - fail if there are changes
+    description: |
+      Chromatic ensures consistency in UI components, down to the pixel.
+      Every commit is automatically tested for visual changes in the cloud.
+      Chromatic - https://www.chromaticqa.com/
+      Orb source code - https://github.com/waveaccounting/orb-chromatic
+
+      Note that this only support yarn for now
+    examples:
```

Comment:
I'm new to orbs for circleci, but do we need these examples? I see that this is from the orb source (https://circleci.com/orbs/registry/orb/wave/chromatic#orb-source) just curious if we can pare it down.

Reply:
no, but this is a straight copy from the orb source, which i'm hoping i'll be able to contribute to so we can completely remove all of this after that happens